.github/workflows/grafana.yml

```diff
@@ -25,13 +21,10 @@ jobs:
     # The secret GRAFANA_URL must be set with the format http://username:password@url.org without a trailing /
     - name: Import dashboards to grafana
       run: >
-        dashboard_list=($(echo $DASHBOARDS));
-        for path in "${dashboard_list[@]}"; do
-        full_path="rendered/${path}";
-        echo "Importing ${full_path}";
-        dashboard=$(cat ${full_path});
+        for t in rendered/**/*.json; do
+        echo "Importing ${t}";
+        dashboard=$(cat ${t});
         echo "{\"dashboard\": ${dashboard}, \"overwrite\": true}" |
         curl -k -Ss -XPOST -H "Content-Type: application/json" -H "Accept: application/json" -d@-
```

Comment:
The internal grafana won't be accessible from outside

Reply:
Yes, Those actions in workflows are not of any use right now. They just help us to do some sanity checks, if we are able to build jsons and deploy them into grafana. To start with lets keep it that way and our plan is to extend the same actions for our automatic deployments in future.

Reply:
But this workflow is gonna fail, is that expected?